tests/rhui3_tests/test_client_management.py

```diff
@@ -5,6 +5,10 @@
 
 from os.path import basename
 
+# The parts related to Atomic can be skipped because they only work on RHEL 7; BZ 1405083.
+import platform
+oldrhel = float(platform.linux_distribution()[1]) < 7
```

Comment:
@RadekBiba, what about a different naming for 'oldrhel'? technically, 7.2 is old too =D. I don't feel it's PC to call RHEL6 'old', it's just an 'older' version, which is still supported.